src/crimson/osd/object_context.h

```diff
@@ -123,8 +123,7 @@ class ObjectContext : public Blocker,
     case RWState::RWNONE:
       return seastar::make_ready_future<>();
     default:
-      ceph_abort_msg("invalid lock type");
```

Comment:
BTW: `ceph_abort_msg()` should be already marked with `[[noreturn]]`. If compiler ignores that, then it's pity.